scripts/tplink-mkimage-2023.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python3
+
+'''A program for manipulating tplink2023 images.
```

Comment:
`tplink2022` was my sarcastic jab at TP-Link's ever changing firmware format. This seems very similar to the 2022 format. I wonder if there's a way to unify the two tools.

At any rate, I recommend running this through `pylint` and addressing any issues.

Reply:
Ran pylint and fixed the issues